src/main/java/us/kbase/auth/AuthConfig.java

```diff
@@ -0,0 +1,120 @@
+package us.kbase.auth;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/** The configuration class for {@link ConfigurableAuthService}.
+ * 
+ * Changes to the config have no effect after {@link ConfigurableAuthService} is instantiated.
+ * 
+ * @author gaprice@lbl.gov
+ *
+ */
+public class AuthConfig {
+	
+	private static final String DEFAULT_KBASE_AUTH_SERVER_URL =
+			"https://ci.kbase.us/services/auth/";
+	
+	private static final String LEGACY_PATH = "api/legacy/KBase/";
+	private static final String LOGIN_LOC = "Sessions/Login/";
+			
+	private URI authServerURL;
+	private boolean allowInsecureURLs = false;
+	
+	/** Get the default authorization URL.
+	 * @return the default authorization URL.
+	 */
+	public static URL getDefaultAuthURL() {
+		try {
+			return new URL(DEFAULT_KBASE_AUTH_SERVER_URL);
+		} catch (MalformedURLException e) {
+			throw new RuntimeException("The impossible just happened");
```

Comment:
If impossible, why use try catch? It an exception could be raised, maybe a more descriptive exception message?

Reply:
Because I don't want to pollute the method signature with a checked exception that can never be thrown